spikeextractors/extractors/yassextractors/__init__.py

```diff
@@ -0,0 +1 @@
+from .yassextractors import YassExtractor
```

Comment:
```suggestion
from .yassextractors import YassSortingExtractor
```

This is failing the tests

Reply:
Ok, travis passes now..